.github/workflows/ci.yml

```diff
@@ -90,6 +90,28 @@ jobs:
     - name: Verify codegen
       run: make verify-codegen
 
+  verify-manifests:
+    needs:
+    - verify-go-modules
+    runs-on: ubuntu-latest
+    steps:
+    - name: Check out code
+      uses: actions/checkout@v4
+    - name: Install Go
+      uses: actions/setup-go@v5
+      with:
+        go-version: ${{ env.GO_VERSION }}
+        cache: true
+        cache-dependency-path: '**/go.sum'
+    - name: Create kind cluster
+      run: make kind-up
+      env:
+        KIND_IMAGE: kindest/node:v1.31.1
+    - name: Verify the WCP manifests
+      run: make verify-wcp-manifests
+    - name: Destroy kind cluster
+      run: make kind-down
```

Comment:
Should we run this step regardless of whether the previous `Verify the WCP manifests` step fails, to ensure the kind cluster is destroyed? Or even better, we could run it as long as the `Create kind cluster` step succeeds, something like `if: ${{ steps.create_kind_cluster.outcome == 'success' }}`.

Reply:
Technically the step is not required at all since the agent goes away right after, but I'll add your suggestion so things are explicit.